src/homework4/task7.py

```diff
@@ -0,0 +1,14 @@
+"""Оглянемся назад. Числа
+   Даны два натуральных числа. Вычислите их наибольший общий делитель
+   при помощи алгоритма Евклида (мы не знаем функции и рекурсию).
+"""
+
+
+a = 15
+b = 100
+while a != 0 and b != 0:
+    if a > b:
+        a = a % b
+    else:
+        b = b % a
+print(a + b)
```

Comment:
It's a cheat - you just don't know where is result (second var is 0).
There is solution without this crutch